app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -112,7 +112,6 @@ private void listenToModelChanges() {
 
         attendeeListener.getNotifier()
             .filter(attendeeModelChange -> attendeeModelChange.getAction().equals(BaseModel.Action.UPDATE))
-            .distinctUntilChanged()
```

Comment:
Using `distinctUntilChanged` filter out subsequent check ins update after first check in, irrespective of identity.
Why do we need it. Does our listener sends same modelChange more than once ?